### ISSUE TITLE
Berstein-Vazirani: change "a" for "s" in 1.3 explanation

### ISSUE THREAD

**Issue by EmilianoG-byte** [Qiskit/textbook#54]
**Is your suggestion related to a problem? Please describe.**
In section 1.3 of [Berstein-Vazirani](https://learn.qiskit.org/course/ch-algorithms/bernstein-vazirani-algorithm) there is an inconsistency when explaining the input to the oracle $f_s$, as in the second to last equation it shows as if we want to obtain $a$, even though this was referred to as the initial state, and the string we want to find out is rather $s$.

This in my opinion makes it a big hard to follow. In the examples that follow, again the string is referred as $s$ and not $a$.

**Describe the solution you'd like**
In the last two equations of section 1.3, the $a$'s should be changed for $s$'s.

**Describe alternatives you've considered**
NA

**Comment by EmilianoG-byte** [comment]
I'd would gladly work on this issue.

**Comment by frankharkins** [comment]
Thanks for making this issue, sorry I missed this. The textbook is being superseded so this will not be fixed.
